packages/components/src/components/hds/rich-tooltip/bubble.ts

```diff
@@ -6,12 +6,32 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
+import type { HdsPopoverPrimitiveSignature } from '../popover-primitive';
+import type { FloatingUIOptions } from '../../../modifiers/hds-anchored-position.ts';
 import {
   DEFAULT_PLACEMENT,
   PLACEMENTS,
 } from '../../../modifiers/hds-anchored-position.ts';
 
-export default class HdsRichTooltipBubbleComponent extends Component {
+export interface HdsRichTooltipBubbleSignature {
+  Args: {
+    placement?: FloatingUIOptions['placement'];
+    offset?: FloatingUIOptions['offsetOptions'];
+    enableCollisionDetection?: FloatingUIOptions['enableCollisionDetection'];
+    width?: string;
+    height?: string;
+    isOpen?: boolean;
```

Comment:
Looks like the docs are missing some arguments

Reply:
Oh, strange! Let me have a look 👀

Reply:
Ah yes, now I see: in some cases, when we don't want consumers to use certain argument for the yielded sub-components (because are [provided directly by the parent component](https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/rich-tooltip/index.hbs#L25-L28)) we omit them from the API documentation

Reply:
Ah, ok. That makes sense. Thanks for clarifying.